### ISSUE TITLE
Extend base game with a part of the MoD features

### ISSUE THREAD

**Issue by gereon77** [Longwelwind/swords-and-ravens#161]
- [x] Sea orders M-1 and Support
- [ ] House Arryn
- [ ] Vale and Narrow Sea overlays for House Arryn
- [x] Make power tokens a currency which allows trading with them
- [x] Start with 7 power tokens

**Comment by Longwelwind** [comment]
**Note:** A player can refuse a gift of Power Tokens.

**Comment by phi1-h** [comment]
Have a look at these [resources](https://git.sr.ht/~phi1_h/GoT_Boardgame/tree/master/static), I made a map with essos and larger fonts, alternative house icons and also order tokens for the MoD expansion.

**Comment by Longwelwind** [comment]
@phi1-h Thanks a lot!

I'm curious, I see source code in the website you've linked. Have you tried to implement MoD yourself? How far were you from a completed system? Were there roadblocks you encountered while trying to implement MoD?

**Comment by phi1-h** [comment]
Yes I did more than half a year ago, but it's unfinished. However I did get a couple of things working, like the map, the order tokens, units with drag and drop to name a few. I worked with sapper(Svelte), mongodb, appollo(graphql). If I remember correctly I got the basic three phases working. But there are many design choices which are flawed I guess. So I don't think I'll continue any work there. But feel free to ask questions and there may be some interesting things already implemented.
